code/game/machinery/cryopod.dm

```diff
@@ -689,6 +696,7 @@
 		to_chat(usr, "<span class='notice'>[on_enter_occupant_message]</span>")
 		to_chat(usr, "<span class='boldnotice'>If you ghost, log out or close your client now, your character will shortly be permanently removed from the round.</span>")
 		occupant = usr
+		begin_processing()
```

Comment:
Side rant: I really hate these move in and move out verbs on these cryopods and sleepers and other things, I wish I could just remove them because I've never once used them.